pkg/hvutils/hvutils.go

```diff
@@ -15,11 +15,17 @@ limitations under the License.
 */
 
 // Package hvutils implements helper functions for the HV API.
+// This package is a layer between the Reconcile methods of the controllers and
```

Comment:
We will need to update this text after updating the functions in this package

Reply:
fixed.